src/iexamine.cpp

```diff
@@ -4559,6 +4562,26 @@ void iexamine::autodoc( player &p, const tripoint &examp )
             }
             break;
         }
+        case TYMPANOPLASTY: {
+            if( !patient.has_effect( effect_deaf ) ) {
+                popup_player_or_npc( patient, _( "Your hearing is normal." ),
+                                     _( "%1$s has normal hearing." ) );
```

Comment:
There is no argument that can be filled in for the "%1$s".
```suggestion
                                     _( "<npcname> has normal hearing." ) );
```

Reply:
Well, either it doesn't work in the earlier code where i've copied the pattern from or it in fact does ;)

Reply:
Referring to an non-existing argument will yield an error string (no crash or similar, if you mean that, just not the expected string).

> in the earlier code where i've copied the pattern from 

Do you have path and line number of something similar?

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/33548/files#diff-3196f85e7ab533f3659c29eedb38d09cR4559

Reply:
WTF! You're right. That popup function works like this, your code here is using it correctly. Some much about consistency. Who needs that anyway. We can just guess and experiment what a function does. So much more fun.